recipes/pyinterp/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "pyinterp" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CNES/pangeo-{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4e2d345c960b7592a01039434d5334d732ad99449ca3bdc5e7075b62c1cc24c1
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cmake
+    - eigen
+    - gsl
+    - libboost
+    - numpy >=1.11
+    - python >=3.6
+    - setuptools
+    - xarray
+  run:
+    - numpy >=1.11
```

if you are compiling against ``numpy`` it should be
```suggestion
    - {{ pin_compatible('numpy') }}
```